docs/reviewer_guide.md

```diff
@@ -0,0 +1,119 @@
+# Reviewer Guide
+
+## Information for New Reviewers
+
+Thank you for accepting the invitation to review a lesson for The Carpentries Lab!
+This document provides guidance to help you with your review.
+
+By accepting the invitation to review a lesson,
+you are agreeing to abide by [The Carpentries Code of Conduct][code-of-conduct].
+Please ensure that you are familiar with the Code of Conduct,
+and read the information in this section before you start your review.
+
+### The Purpose of Lesson Review
+
+The goal of The Carpentries Lab lesson review process is to improve the lesson.
+Lessons approved for inclusion in The Carpentries Lab should be ready to teach,
+by any certified Carpentries Instructor with sufficient knowledge of
+the domain and/or topic of the lesson.
+For this reason, the focus of the lesson review process is on
+the content of the lesson and the supporting material that can be used by
+learners and Instructors before, during, and after a workshop.
+
+### Review Scope
+
+When making your review,
+read through the lesson homepage and all the episode pages,
+as well as the pages under the Extras dropdown (_Setup_, _Instructor Notes_, etc).
+You should make a copy of the [Reviewer Checklist](#reviewer-checklist) below to guide your review,
+but are also welcome to make comments and request changes outside of the points listed there.
+In particular, we encourage you to make some general comments about
+
+- the readability of the lesson.
+- any key concepts or skills relevant to the lesson topic/domain that are missing from the lesson.
+- how it compares to any other learning resources that you are aware of on the same/similar topics.
+- its utility as a resource both for an Instructor teaching the lesson at a workshop
+  and for a self-directed learner following the lesson alone.
+
+The scope of your review is limited to the contents of the lesson website,
+though you may find it helpful or necessary to refer to the source (R)Markdown files
+to address some points in the Reviewer Checklist.
```

Comment:
expand on this, to give example(s) of when this kind of check of the source code might be necessary.